browser-test/src/checkbox.test.ts

```diff
@@ -1,40 +1,23 @@
-import {Page} from 'playwright'
 import {
-  AdminPrograms,
-  AdminQuestions,
-  ApplicantQuestions,
+  createTestContext,
   loginAsAdmin,
   loginAsGuest,
   logout,
   selectApplicantLanguage,
-  startSession,
-  resetSession,
   validateAccessibility,
   validateScreenshot,
 } from './support'
 
 describe('Checkbox question for applicant flow', () => {
-  let pageObject: Page
-
-  beforeAll(async () => {
-    const {page} = await startSession()
-    pageObject = page
-  })
-
-  afterEach(async () => {
-    await resetSession(pageObject)
-  })
+  const ctx = createTestContext(/* clearDb= */ false)
 
   describe('single checkbox question', () => {
-    let applicantQuestions: ApplicantQuestions
     const programName = 'test program for single checkbox'
 
     beforeAll(async () => {
```

Comment:
not-blocking, and no action needed. just for discussion - In theory, we should be able to just change this to beforeEach for all the test files and set clearDB=true and that should work right? Do you know if there's a large performance hit?

Reply:
Yes, replacing `beforeAll` with `beforeEach` will allow us to clean DB between tests. But as you said it does introduce performance hit and pretty nasty :/ 

I just wrote down some thoughts about that here: https://github.com/civiform/civiform/issues/2891#issuecomment-1235813976